ibctest/relayer.go

```diff
@@ -0,0 +1,256 @@
+package ibctest
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/relayer/v2/cmd"
+	"github.com/cosmos/relayer/v2/internal/relayertest"
+	"github.com/cosmos/relayer/v2/relayer/provider/cosmos"
+	"github.com/ory/dockertest/v3"
+	"github.com/strangelove-ventures/ibctest/ibc"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest"
+)
+
+type Relayer struct {
+	t *testing.T
+
+	pool *dockertest.Pool
```

Comment:
`pool` is now unused and can be removed.

Reply:
removed